### ISSUE TITLE
Can't build version 2.X with Webpack (module has no exports)

### ISSUE THREAD

**Issue by boldtrn** [maplibre/maplibre-gl-js#370]
I just tried to upgrade from 1.5.2 to 2.0.0-pre.4.

However I am facing issue during the build with Webpack.

With 1.5.2 the following works:
```
import * as maplibregl from 'maplibre-gl';
new maplibregl.LngLatBounds([-180, -85.0511], [180, 85.0511]);
```

However, with 2.0.0-pre.4 I get the following error during the Webpack build:

```
export 'LngLatBounds' (imported as 'maplibregl') was not found in 'maplibre-gl' (module has no exports)
```

I am using JS and no TS - not sure if there might be issues with this?

**Comment by HarelM** [comment]
Please provide a jsbin/stackblitz/something else with the full setup.

**Comment by boldtrn** [comment]
I just did a quick test on Stackblitz, it works there. So maybe it's related to Webpack. I could try to setup a small repo for reproduction. Webpack is always a bit of a overhead ;)

**Comment by HarelM** [comment]
This is my impression as well, might be something we changed in 2.0, but I'm not sure...
Webpack configuration is not easy... Let me know if you find the root cause.

**Comment by boldtrn** [comment]
Ok, I was able to identify the issue. The problem is created by a [line in the package.json](https://github.com/maplibre/maplibre-gl-js/blob/main/package.json#L16).

```
"type": "module",
```

When I remove the line, the build works.

So this seems to be at least related to https://github.com/maplibre/maplibre-gl-js/pull/368

**Comment by HarelM** [comment]
Also related to #334 I think.
In any case I now see this issue too when I build using webpack. I'll see what I can find in google...

**Comment by HarelM** [comment]
I did a small test where I added a `package.json` file to the dist folder inside the `node_modules` in this file all I wrote was:
`{ type: "commonjs" }` and this seems to solve this issue I believe.
Since the library is exported as commonJs and not esm adding this file as part of the build shouldn't be a big issue I believe...

**Comment by HarelM** [comment]
@boldtrn Can you check if this is fixed in version 2.0.0-pre.6? If so can you close this issue?

**Comment by boldtrn** [comment]
Thanks a lot for fixing this issue, I just double check and can confirm that the build works 🚀
